src/main/java/io/aiven/klaw/controller/EnvsClustersTenantsController.java

```diff
@@ -167,15 +167,6 @@ public ResponseEntity<List<EnvModel>> getKafkaConnectEnvs() {
         envsClustersTenantsControllerService.getKafkaConnectEnvs(), HttpStatus.OK);
   }
 
-  @RequestMapping(
-      value = "/getSchemaRegEnvsStatus",
-      method = RequestMethod.GET,
-      produces = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<List<EnvModel>> getSchemaRegEnvsStatus() {
-    return new ResponseEntity<>(
-        envsClustersTenantsControllerService.getSchemaRegEnvsStatus(), HttpStatus.OK);
-  }
-
```

Comment:
Since it is an endpoint which could be used by users should the migration process somehow be described anywhere?
Like what to use instead after upgrading to Klaw 1+?

Reply:
This is an unused endpoint which is being removed here. It is not called from anywhere. There was some code related to PLAINTEXT protocol and hence found. It's just part of cleanup.